codespell_lib/data/dictionary.txt

```diff
@@ -21780,11 +21790,20 @@ overlaping->overlapping
 overlapp->overlap
 overlayed->overlaid
 overlflow->overflow
+overlflowed->overflowed
+overlflowing->overflowing
+overlflows->overflows
 overlfow->overflow
+overlfowed->overflowed
+overlfowing->overflowing
+overlfows->overflows
 overlodaded->overloaded
 overloded->overloaded
 overlodes->overloads
 overlow->overflow
+overlowed->overflowed
```

Comment:
Possibly overloaded at a stretch.

Reply:
Pushed a new version of the PR with this suggestion applied.